test/format/verify.c

```diff
@@ -389,6 +404,8 @@ table_verify_mirror(
         if (last_failures == failures && (base->type == ROW || table->type == ROW))
             last_match = base_keyno;
     }
+
+done:
```

Comment:
We need to close this new cursor before returning.

Reply:
Never mind. We are closing the session completely here that will close the cursor.